src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -1044,6 +1068,179 @@ components:
           type: string
           description: A specific type of in-prison education/training that does not fit the given 'inPrisonEducation' types. Mandatory when 'inPrisonEducation' includes 'OTHER'.
 
+    UpdateCiagInductionRequest:
+      title: UpdateCiagInductionRequest
+      description: A request to update a CIAG Induction. Migrated from hmpps-ciag-careers-induction-api.
+      type: object
+      properties:
+        reference:
+          type: string
+          format: uuid
+          description: The Induction's unique reference. Currently optional (to avoid breaking the existing CIAG API), but will be mandatory in future (and will most likely become a path variable).
+          example: c88a6c48-97e2-4c04-93b5-98619966447b
+        prisonId:
+          type: string
+          description: The ID of the Prison that that Prisoner is currently at.
+        hopingToGetWork:
+          $ref: '#/components/schemas/HopingToWork'
+        reasonToNotGetWorkOther:
+          type: string
+          description: The reason that is given when the Prisoner does not want to work. This is mandatory when 'reasonToNotGetWork' is set to 'OTHER'.
+        abilityToWork:
+          uniqueItems: true
+          type: array
+          description: One or more factors affecting the Prisoner's ability to work.
+          items:
+            $ref: '#/components/schemas/AbilityToWorkFactor'
+        abilityToWorkOther:
+          type: string
+          description: A specific factor affecting the Prisoner's ability to work. This is mandatory when 'abilityToWork' is set to 'OTHER'.
+        reasonToNotGetWork:
+          uniqueItems: true
+          type: array
+          description: One or more reasons the Prisoner has given not to get work after leaving Prison.
+          items:
+            $ref: '#/components/schemas/ReasonNotToWork'
+        workExperience:
+          $ref: '#/components/schemas/UpdatePreviousWorkRequest'
+        skillsAndInterests:
+          $ref: '#/components/schemas/UpdateSkillsAndInterestsRequest'
+        qualificationsAndTraining:
+          $ref: '#/components/schemas/UpdateEducationAndQualificationsRequest'
+        inPrisonInterests:
+          $ref: '#/components/schemas/UpdatePrisonWorkAndEducationRequest'
+      required:
+        - hopingToGetWork
+        - prisonId
+
+    UpdatePreviousWorkRequest:
+      description: A request to persist the previous work experience of the Prisoner. Migrated from hmpps-ciag-careers-induction-api.
+      type: object
+      properties:
+        id:
```

Comment:
This is the only difference from the existing "Create" request objects.

Thinking about it, I could consider using `allOf` here to reduce some of the duplication 🤔

Reply:
Yeah, sounds like a good idea 👍